main.py

```diff
@@ -1,10 +1,40 @@
+import json
 import init_django_orm  # noqa: F401
-
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for nickname, player_data in players_data.items():
```

Comment:
It's a good practice to validate the structure and content of 'players_data' before processing it. This can prevent runtime errors if the JSON structure is not as expected.